core/rpc_client.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
```

Comment:
Ah this must be a leftover from the olden days where these files lived directly in the top level directory and were scripts to execute directly.